canonical-web-and-design/ubuntu.com
# Add buy.u.c takeover

## Done

- Added buy.u.c takeover
- Removed enterprise snap mgmt takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the text matches option three in the [brief](https://docs.google.com/document/d/1DKJGXSyB5gooLxM_hSmuydm3sbQu4h3jtND4DauQ9sk/edit?ts=5d6fb58b#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1638

## Screenshots

![image](https://user-images.githubusercontent.com/441217/64429161-4c53f100-d0ad-11e9-911b-72a29d122500.png)
